custom/covid/management/commands/update_cases.py

```diff
@@ -42,6 +42,15 @@ def handle(self, domain, case_type, **options):
         else:
             user_id = SYSTEM_USER_ID
 
+        try:
+            if options["location"]:
+                active_location = options["location"]
+            else:
+                print("Warning: No active location was entered")
```

Comment:
i dont think you need to print a warning here. especially since two of the commands that use this do not even accept it as an argument.